sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_mldesigner/__init__.py

```diff
@@ -13,7 +13,9 @@
 from azure.ai.ml.entities._component.component_factory import component_factory
 from azure.ai.ml.entities._job.pipeline._load_component import _generate_component_function
 from azure.ai.ml.entities._inputs_outputs import _get_param_with_standard_annotation
-from azure.ai.ml._internal.entities._additional_includes import _AdditionalIncludes  # pylint: disable=unused-import
+from azure.ai.ml._internal.entities._additional_includes import (
+    _InternalAdditionalIncludes,
+)  # pylint: disable=unused-import
```

Comment:
why this. By the way, the pylint disable can be removed after adding `_InternalAdditionalIncludes` to `__all__`

Reply:
Not sure why add this before, but not import. I will add it to __all__

Reply:
I will add it to __all__

Reply:
it seems also need open a pr for mldesigner according to: https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2?path=/docs/mldesigner/imports_from_azure_ai_ml.md&version=GBmain&_a=preview